nservicebus/messaging/send-a-message_immediate-dispatch_core_[6.0,].partial.md

```diff
@@ -1,4 +1,11 @@
+The preferred approach is to use the the explicit immediate dispatch API:
 
-The preferred approach is to use the the explicit immediate dispatch API.
+snippet:RequestImmediateDispatch
 
-snippet:RequestImmediateDispatch
\ No newline at end of file
+WARNING: By specifying immediate dispatch, outgoing messages will not be [batched](/nservicebus/messaging/batched-dispatch.md) or enlisted in the current receive transaction even if the transport has support for it.
+
+Suppressing the ambient transaction to have the outgoing message sent immediately is also possible:
```

Comment:
@timbussmann - which one? Immediate dispatch or suppressing the ambient transaction?